### PR TITLE
Update cd.yml

### PR DESCRIPTION
**Pull request by chronark**

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -29,4 +29,18 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           push: true
-          tags: perfolio/api-data:latest
+          tags: perfolio/api-data:lates
```
Comment (webersni) [inline .github/workflows/cd.yml:32]: typo

Comment (webersni) [reply .github/workflows/cd.yml:32]: latest

**Comment by webersni** [comment]
@chronark 
There's a small typo.

File: .github/workflows/cd.yml (hunk 1 above)
Comment (chronark) [reply .github/workflows/cd.yml]: no :D

**Pull request merged**